frontend/src/Editor/Container.jsx

```diff
@@ -296,11 +299,15 @@ export const Container = ({
     const boundingRect = document.getElementsByClassName('canvas-area')[0].getBoundingClientRect();
     const canvasWidth = boundingRect?.width;
 
-    width = Math.round(width + (deltaWidth * 43) / canvasWidth); // convert the width delta to percentage
+    width = Math.round(width + (deltaWidth * NO_OF_GRIDS) / canvasWidth); // convert the width delta to percentage
     height = height + deltaHeight;
 
     top = y;
-    left = (x * 100) / canvasWidth;
+    if (direction === 'left') {
+      // onResizeStop is triggered for a single click on the border, therefore this conditional logic
+      // should not be removed.
+      left = (x * 100) / canvasWidth;
+    }
```

Comment:
```suggestion
    left = (x * 100) / canvasWidth;
```

Even here the check is not needed

Reply:
Hi Kavin, Yes feature works without the checks. Added these conditions checks since in these cases the left position of the widget is not expected to change. But we can remove them.